dash/_validate.py

```diff
@@ -201,7 +201,7 @@ def validate_multi_return(outputs_list, output_value, callback_id):
                     dedent(
                         f"""
                         Invalid number of output values for {callback_id} item {i}.
-                        Expected {len(vi)}, got {len(outi)}
+                        Expected {len(outi)}, got {len(vi)}
```

Comment:
Thanks for catching this.

Look like these were inverted by mistake, probably because the variable name is not specific enough. Do you think you can also change the name of the `outi` and `vi` variables to `output_spec` and `output_value` like the error message ?